<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1619</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QlineEdit {
rgb(144, 203, 255)
}</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="positionSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>720</y>
      <width>811</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="play_push">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_frame_push">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>760</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_frame_push">
    <property name="geometry">
     <rect>
      <x>424</x>
      <y>760</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>750</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; vertical-align:super;&quot;&gt;Offset&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="videoWindow">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>1061</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_offset">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>770</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>1591</width>
      <height>431</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>40</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="fps_spin_box">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>770</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>750</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="y_axes_range">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>750</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Y axes range</string>
    </property>
    <widget class="QSpinBox" name="spinBox_max">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>29</y>
       <width>35</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-999999999</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>35</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>35</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-99999999</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>35</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="spinBoxGoFrame">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>770</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>750</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Go to frame</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>750</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Go to time</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="go_to_time_spin">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>770</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject video (time/frame)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>160</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stimulus Video (Time/frame)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_stim">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>180</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>110</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stimulus number</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1619</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData_Visualizer">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Video"/>
    <addaction name="actionLoad_data"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRaw_data"/>
    <addaction name="actionPreprocessed_data"/>
    <addaction name="actionICA"/>
    <addaction name="actionRMS"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="menuNavigation_type">
     <property name="title">
      <string>Navigation type</string>
     </property>
     <addaction name="actionFrames"/>
     <addaction name="actionTime_of_signal"/>
    </widget>
    <widget class="QMenu" name="menuDown_sampling_value">
     <property name="title">
      <string>Down sampling value</string>
     </property>
     <addaction name="actionvalue"/>
     <addaction name="actionauto"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionFPS"/>
    <addaction name="menuNavigation_type"/>
    <addaction name="menuDown_sampling_value"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuSegmentation">
     <property name="title">
      <string>Segmentation </string>
     </property>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuSegmentation"/>
    <addaction name="actionSlice_navigator"/>
    <addaction name="actionTag"/>
   </widget>
   <addaction name="menuData_Visualizer"/>
   <addaction name="menuView"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Video">
   <property name="text">
    <string>Load Video</string>
   </property>
  </action>
  <action name="actionLoad_data">
   <property name="text">
    <string>Load data</string>
   </property>
  </action>
  <action name="actionRaw_data">
   <property name="text">
    <string>Raw data</string>
   </property>
  </action>
  <action name="actionPreprocessed_data">
   <property name="text">
    <string>Preprocessed data</string>
   </property>
  </action>
  <action name="actionICA">
   <property name="text">
    <string>ICA </string>
   </property>
  </action>
  <action name="actionFPS">
   <property name="text">
    <string>FPS</string>
   </property>
  </action>
  <action name="actionFrames">
   <property name="text">
    <string>Video frames</string>
   </property>
  </action>
  <action name="actionTime_of_signal">
   <property name="text">
    <string>Time of signal</string>
   </property>
  </action>
  <action name="actionvalue">
   <property name="text">
    <string>Value</string>
   </property>
  </action>
  <action name="actionauto">
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionRMS">
   <property name="text">
    <string>RMS</string>
   </property>
  </action>
  <action name="actionMaximum_number_of_elements">
   <property name="text">
    <string>Maximum number of elements</string>
   </property>
  </action>
  <action name="actionSlice_navigator">
   <property name="text">
    <string>Slice navigator</string>
   </property>
  </action>
  <action name="actionTag">
   <property name="text">
    <string>Tag</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
